pages.ca/windows/del.md

```diff
@@ -0,0 +1,32 @@
+# del
```

Comment:
I think you might have picked the wrong directory here.

Reply:
Correct. Didn't notice. Probably mis-clicked on the website in the table 😅 Thanks!